protoc-gen-openapiv2/options/openapiv2.proto

```diff
@@ -175,6 +175,41 @@ message Operation {
   // extra functionality that is not covered by the standard OpenAPI Specification.
   // See: https://swagger.io/docs/specification/2-0/swagger-extensions/
   map<string, google.protobuf.Value> extensions = 13;
+  // Custom parameters such as HTTP request headers.
+  Parameters parameters = 14;
+}
+
+// `Parameters` is a representation of OpenAPI v2 specification's parameters object.
+message Parameters {
+  // `Headers` is one or more HTTP header parameter.
+  repeated HeaderParameter headers = 1;
```

Comment:
Note that I believe this is the first instance of these definitions breaking from the general OpenAPI 2 definition structure. I think it's okay in this case since we do want to limit what users can put in here. For example, I don't think we ever want to support users specifying custom  non-header parameters beyond those inferred from the Protobuf schema.

Reply:
I'll add a note underneath.

Reply:
I've added a little note about this 🙌